elide-graphql/src/test/resources/graphql/responses/update/updateComplexAttributeMap.json

```diff
@@ -8,7 +8,7 @@
             {
               "key": "2016-01-02T00:00Z",
               "value": {
-                "units": 126.0,
+                "units": 490056468,
```

Comment:
Why did this value change?

Reply:
Wanted to put in some big value to test the change. Prior to this change, this big value will be returned something as 4.9E. With this new change it does not return exponential value and return the big value as is.